File: Projects/leetcode/MaximumWealth.java
package leetcode;

import java.util.Arrays;
import java.util.function.ToIntFunction;

class MaximumWealth {
    private static ToIntFunction<int[]> sum = account ->
        Arrays.stream(account).sum();

    public static int maximumWealth(int[][] accounts) {
        return Arrays.stream(accounts).mapToInt(sum).max().getAsInt();
    }
}

File: Projects/leetcode/maximumWealth.kt
package leetcode

import java.util.function.ToIntFunction
import java.util.Arrays

private val sum = ToIntFunction { account: IntArray? -> Arrays.stream(account).sum() }

fun maximumWealth(accounts: Array<IntArray>?): Int {
    return Arrays.stream(accounts).mapToInt(sum).max().asInt
}
